search-sentiments-map/public/country-name.json

```diff
@@ -0,0 +1,179 @@
+[
```

Comment:
Did you rename this file from country-code to country-name?

Reply:
No I created a new file. This has the same data as country code, but is instead sorted alphabetically by country name(rather than country code, as the country-code.json file is). I use this file to create the select drop down of all the countries, and so we need them to be sorted alphabetically by name.

Reply:
gotcha